core/src/main/java/feign/Response.java

```diff
@@ -37,298 +37,315 @@
  */
 public final class Response implements Closeable {
 
-  private final int status;
```

Comment:
could you revert this format changes?

runninng `mvn clean install` should fix code format

Reply:
done